server.js

```diff
@@ -1,60 +1,121 @@
-require('dotenv').config();
-const express = require('express');
-const exphbs = require('express-handlebars');
-const cookieParser = require('cookie-parser');
-const morgan = require('morgan');
-const passport = require('passport');
-const moment = require('moment');
-const helmet = require('helmet');
+require("dotenv").config();
+const express = require("express");
+const exphbs = require("express-handlebars");
+const cookieParser = require("cookie-parser");
+const morgan = require("morgan");
+const passport = require("passport");
+const moment = require("moment");
+const helmet = require("helmet");
 const PORT = process.env.PORT || 3333;
 const app = express();
-const db = require('./models');
+const db = require("./models");
 
 app.use(cookieParser());
 app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
-app.engine('handlebars', exphbs({ defaultLayout: 'main' }));
-app.set('view engine', 'handlebars');
+app.engine("handlebars", exphbs({ defaultLayout: "main" }));
+app.set("view engine", "handlebars");
 
-if (app.get('env') !== 'test') {
-  app.use(morgan('dev')); // Hook up the HTTP logger
+if (app.get("env") !== "test") {
+  app.use(morgan("dev")); // Hook up the HTTP logger
 }
 
-app.use(express.static('public'));
+app.use(express.static("public"));
 
-require('./config/passport')(db, app, passport); // pass passport for configuration
+require("./config/passport")(db, app, passport); // pass passport for configuration
 
 // Define our routes
-app.use('/api', require('./routes/apiRoutes')(passport, db));
-app.use(require('./routes/htmlRoutes')(db));
+app.use("/api", require("./routes/apiRoutes")(passport, db));
+app.use(require("./routes/htmlRoutes")(db));
 
 // Secure express app
-app.use(helmet.hsts({
-  maxAge: moment.duration(1, 'years').asMilliseconds()
-}));
+app.use(
+  helmet.hsts({
+    maxAge: moment.duration(1, "years").asMilliseconds(),
+  })
+);
 
 // catch 404 and forward to error handler
-if (app.get('env') !== 'development') {
+if (app.get("env") !== "development") {
   app.use((req, res, next) => {
-    const err = new Error('Not Found: ' + req.url);
+    const err = new Error("Not Found: " + req.url);
     err.status = 404;
     next(err);
   });
 }
 
 const syncOptions = {
-  force: process.env.FORCE_SYNC === 'true'
+  force: process.env.FORCE_SYNC === "true",
 };
 
-if (app.get('env') === 'test') {
+if (app.get("env") === "test") {
   syncOptions.force = true;
 }
 
 db.sequelize.sync(syncOptions).then(() => {
-  if (app.get('env') !== 'test' && syncOptions.force) {
-    require('./db/seed')(db);
+  if (app.get("env") !== "test" && syncOptions.force) {
+    require("./db/seed")(db);
   }
 
+  // Map stuff starts here
```

Comment:
I do not think we should push all this commented out code to the main server?

Reply:
Agreed. How can I fix it?

Reply:
will need to cancel this pull request and fix any of these changes you would want to make.